CHANGELOG.md

```diff
@@ -9,12 +9,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Added ScrollLineUp and ScrollLineDown actions for scrolling line by line
-- Added ToggleFullscreen action (macos also has a ToggleSimpleFullscreen action
-    which allows switching to fullscreen without occupying another space)
+- Added ToggleFullscreen action
+- Added ToggleSimpleFullscreen action for macos which allows switching to fullscreen 
+    without occupying another space
```

Comment:
I'd recommend this to move the `macOS` mention to the front, since that way non-macos users don't have to read through half of it before they realize that it doesn't affect them.

I think some other entries in our change log have used `On macOS, ...` for this.